itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/TestCompactor.java

```diff
@@ -398,7 +398,7 @@ public void testStatsAfterCompactionPartTbl() throws Exception {
             .getParameters();
     Assert.assertEquals("The number of files is differing from the expected", "2", parameters.get("numFiles"));
     Assert.assertEquals("The number of rows is differing from the expected", "2", parameters.get("numRows"));
-    Assert.assertEquals("The total table size is differing from the expected", "1396", parameters.get("totalSize"));
+    Assert.assertEquals("The total table size is differing from the expected", "1395", parameters.get("totalSize"));
```

Comment:
Are you sure that these size changes are relevant? Number of rows, or files are ok. But the total size just did not give me any information. I think it should be removed with the PR. It is just too many works for an Assert that is constantly change.

Reply:
They are changing in every single orc upgrade. Usually with about 1 bytes. As far I remember, it is because the orc file itself contains some version information about orc itself. 
At one of the previous PRs I had the same concern about it as I see no value on those checks. I cannot remember who answered but the answer was something like 'Ok, but what if something goes wrong and there will be a significant change in the size? If we don't check it, we won't know about that'. 
TBH, I'm not completely on board with this answer, I'm pretty sure there are other ways of spotting a significant change in the size of the orc files.

Reply:
I don't see any value to check. A good test is deterministic, to test something that is contantly change it does not make sense.